### PR TITLE
fix(ui5-popover): calculate offset for all sides of the page

### PR DESCRIPTION
**Pull request by gmkv**
The popover has a 10px offset from the edges of the screen,
so that its size would not reach the border of the page.

Left and Top positons are now clamped to adjust for that offset on all sides.

Fixes: #2697

### CODE REVIEW

**Review by fifoosid — DISMISSED** [review]
Looks good from code POV, waiting for a review from @SAP/ui5-webcomponents-topic-rd before we merge it

**Review by TeodorTaushanov — APPROVED** [review]

**Comment by gmkv** [comment]
Rebased to resolve merge confict.

**Review by fifoosid — APPROVED** [review]

**Pull request merged**
